controllers/comments.controller.js

```diff
@@ -0,0 +1,17 @@
+const { insertComment } = require("../models/comments.model");
+
+exports.postComment = async (req, resp, next) => {
+  const article_id = parseInt(req.params.article_id);
+  const comment = {
+    username: req.body.username,
+    body: req.body.body,
+    article_id,
+  };
+  insertComment(comment)
+    .then(async (data) => {
+      resp.status(201).send(data);
```

Comment:
try to send this on a key of 'postedComment' or something like that, honestly for readability more than anything :)